### DISCUSSION TITLE
Download beatmap to all lazer devices from website

### DISCUSSION THREAD

**Discussion by leowla** [ppy/osu#19755] [Ideas] [answered]
It would be great if there was a button on the osu website to download the beatmap to all devices running osu. I have osu on my PC and iPad, so it would save me some hassle if I could download a beatmap to both devices.

The way I imagine it is for all beatmap pages on the website to have a button next to download/osu!direct that when clicked will save the map to a download queue on the account. Then when the user opens osu on a device with that account, it checks if there are any maps in the queue and automatically downloads them.

**Comment by peppy** [comment, marked as answer]
We may add some kind of synchronisation in the future, but it's not an immediate focus.

If you want to download beatmaps you played recently, you may be able to use the "played" filter at beatmap listing (supporter only).
